crates/sui-core/src/unit_tests/gas_tests.rs

```diff
@@ -548,7 +543,24 @@ async fn execute_transfer_with_price(
     let data =
         TransactionData::new_with_gas_price(kind, sender, gas_object_ref, gas_budget, gas_price);
     let signature = Signature::new(&data, &sender_key);
-    let tx = Transaction::new(data, signature);
+    (Transaction::new(data, signature), authority_state)
+}
+
+async fn execute_transfer_transaction_inner(
```

Comment:
Whats the `_inner` part here? Just don't understand the naming difference between these two functions
But maybe I'm just being a bit dense